### PR TITLE
Symfony 3 support

### PR DESCRIPTION
**Pull request by michailw**
This fork is useful on Symfony 3. Please merge it.

### CODE REVIEW

**Comment by hmert** [comment]
@michailw this is really huge. Thank you for your effort. @APY/collaborators will review it.

Could you solve the conflicts of PR please? Have you tested all?

**Comment by qferr** [comment]
This PR creates a BC break !!!!

**Comment by michailw** [comment]
It's not mine work. I use this fork in my project with Symfony3, but it
doesn't have translation_domain, but I need it. Thats why I ask for merging
this fork to main bundle.
05.04.2016 8:48 PM "qferr" <notifications@github.com> napisał(a):

> This PR creates a BC break !!!!
>

**Comment by qferr** [comment]
It's impossible to merge your PR in the master because the current milestone is 2.3. You could merge `APY:master` to `artscorestudio:master` for get last features and resolve conflicts.

**Comment by hoshsadiq** [comment]
@qferr As I need this bundle to be 3.0 compatible (currently having to settle with 2.8 with this being one of the few things still needing upgrading), I'm happy to fork the repo and resolve the conflicts but I need to know I won't be wasting my time as I've noticed none of the 3.0 backwards incompatible changes have been merged.

**Comment by qferr** [comment]
@hoshsadiq see #849

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "apy/datagrid-bundle",
+    "name": "artscorestudio/datagrid-bundle",
```
Comment (rvanlaarhoven) [inline composer.json]: I noticed you changed the repository name, which should not be included in this PR.

**Comment by yenteck** [comment]
it works fine with symfony 3 ! just use "name": "artscorestudio/datagrid-bundle" in composer.json

**Comment by utkuaydin** [comment]
I have created [a new branch](https://github.com/APY/APYDataGridBundle/tree/symfony3) for this pull request. Necessary modifications such as correction of `composer.json` can be impemented there by anyone. Thanks for the PR!

**Comment by hmert** [comment]
I will close this PR since its contents are in `symfony3` branch. We will merge `symfony3` branch after the work on it is done. I’ll make sure authors will be credited. Thanks!
